core/src/main/resources/hudson/diagnosis/Messages.properties

```diff
@@ -6,5 +6,6 @@ HudsonHomeDiskUsageMonitor.DisplayName=Disk Usage Monitor
 
 NullIdDescriptorMonitor.DisplayName=Missing Descriptor ID
 ReverseProxySetupMonitor.DisplayName=Reverse Proxy Setup
-ReverseProxySetupMonitor.EncodedForwardSlashesDisallowed=Detected that encoded slashes are not allowed.  If you are running Tomcat, please be sure to run it with -Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true
+ReverseProxySetupMonitor.EncodedForwardSlashesDisallowed=Detected that encoded slashes are not allowed.  If you are running Tomcat, please be sure to run it with -Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true\n\
+  more info at: https://wiki.jenkins.io/display/JENKINS/Jenkins+says+my+reverse+proxy+setup+is+broken
```

Comment:
I would recommend linking only to the wiki, as `doAct` did, and keep all container-specific details there.